sensing/pointcloud_preprocessor/launch/distortion_corrector.launch.xml

```diff
@@ -0,0 +1,6 @@
+<launch>
```

Comment:
Remove this file :)

Reply:
@Ariiees 
Probably you need to pull the latest universe https://github.com/autowarefoundation/autoware.universe/blob/main/sensing/pointcloud_preprocessor/launch/distortion_corrector_node.launch.xml

Reply:
Also would be appreciated that let me click the resolved button to make sure it is solved. Thanks!

Reply:
Sure! Thank you for help me checking the file!